backend/graminsta/user/models.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: UTF-8 -*-
+
+"""models.py
+"""
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class UserInfo(models.Model):
+    """UserInfo Class
+    """
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    age = models.IntegerField()
+    gender = models.CharField(max_length=10)
+    reg_time = models.DateTimeField(auto_now=True)
+    following_users = models.ManyToManyField(
+        "self",
+        symmetrical=False,
+        related_name="following_users"
```

Comment:
This is used to do reverse look up.

For example, suppose `related_name="followed_by"', we have user A, B, C.

* User B is following User A
* User C is following User A
* User C is following User B

So for User C: `user_info_for_user_c.following_users == [User A, User B]`
for User A: `user_info_for_user_a.followed_by == [User B, User C]`

See also: https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.ForeignKey.related_name

Reply:
And the follow relationship should be defined on the `User` model, so I'd suggest to define that explicitly.

See also: https://stackoverflow.com/questions/40069192/django-models-database-design-for-user-and-follower

Reply:
And it looks like the follow/followed by relationship is tightly coupled with the `Post` app, so maybe define the relationship in that app would be more reasonable.